jpscore/CMakeLists.txt

```diff
@@ -66,7 +66,34 @@ install(TARGETS jpscore
         COMPONENT applications)
 
 set(CT_DATA_FILE_DIR "demos")
-file(GLOB CT_FILES "${CMAKE_SOURCE_DIR}/${CT_DATA_FILE_DIR}/*/*")
+set(CT_FILES
```

Comment:
Hey, why do you write these file explicitly?

Reply:
To make sure that only the files are copied to the install path that are defined. Otherwise all files within the demo directory would be copied even if not intended by us, e.g., the user has added some files to the directory.